src/judging_round_selector.js

```diff
@@ -16,7 +16,7 @@ class JudgingRoundSelector extends React.Component {
     }
 
     fetchJudgingRoundList() {
-	fetch(judgingRoundUrl, {credentials: "include", mode: "cors"})
+	fetch(judgingRoundUrl + "?round_type=Online", {credentials: "include", mode: "cors"})
```

Comment:
Not a blocker, but just as with django, I wish there were better ways to build this sort of URL.